.codegen.json

```diff
@@ -23,11 +23,7 @@
         "post_generate": [
             "go test -timeout 240s -run TestConsistentDatabricksSdkVersion github.com/databricks/cli/internal/build",
             "make schema",
-            "make generate-validation",
-            "echo 'bundle/internal/tf/schema/\\*.go linguist-generated=true' >> ./.gitattributes",
-            "echo 'go.sum linguist-generated=true' >> ./.gitattributes",
```

Comment:
Also removing the annotations for `go.sum` because it's a small file so the diffs should not be too large.